lib/supabase/migrations/20251210110140_add_currency_field_in_payments_table.sql

```diff
@@ -0,0 +1,3 @@
+alter table "public"."in_process_payments" add column "currency" text not null default '0x0000000000000000000000000000000000000000'::text;
```

Comment:
DRY Principle
- why are we storing a duplicate `currency` field in `in_process_payments` rather than referencing the existing `currency` in the `in_process_sales` table?